projects/application/ipfs-souko-en.md

```diff
@@ -0,0 +1,78 @@
+## Project Name
+NonEntropy Japan, IPFS SOUKO　and Joint Research with Graduate School of Information Science and Technology, Osaka University
+
+![logo](https://gyazo.com/eac5a61fcdde999d269721c7ce5b4227.jpeg)
+
+## Category 
+Application, IPFS/Filecoin cloud storage services.
+
+## Project Description
+IPFS SOUKO is a digital archive service that enables smooth uploading and downloading of data between conventional storage (including cloud) and IPFS/Filecoin distributed storage.
+
+IPFS and Filecoin are cutting-edge technologies, and there are still very few people who are proficient in handling them. The simple UI/UX of IPFS SOUKO allows anyone to store (upload) or retrieve (download) their data to IPFS or Filecoin distributed storage.
+
+NonEntropy Japan Launches Joint Research with Osaka University on
+* The Optimization of the IPFS/Filecoin network performance, and labor-saving and data allocation in the distributed network.
+* Security Enhancement to Promote IPFS/Filecoin Distributed Storage
+
+![osaka univ](https://gyazo.com/b3d56626d62d4550ca6ddfa7f1e87d53.jpeg)
+
+## Project Status
+<!--brainstorming, fundraising, under development, beta, shipped, etc-->
+Joint research with Osaka university has started on 1st April, 2021.
+IPFS SOUKO has released the first version, which has completed basic functions on 19th April. More functions are being developed.
+
+
+## Previews
+<!--Add some screenshots to give a preview of your product-->
+![IPFS SOUKO image](https://gyazo.com/748f7b883779a2163332d8442a73e662.jpeg)
+![IPFS SOUKO structure](https://gyazo.com/fda07acd1415e71eed26b429510f6516.jpeg)
+
+## Target Audience
+<!--Describe who will be your project's users-->
+Those who manage digital archive and take care of Filecoin storage system
+
+## Rough estimated user base (if applicable)
+<!--How many users do you have right now?-->
+About 5,000 as of April 2021.
+
+## Github repo
+<!--Attach a link to your GitHub repo if it's OSS-->
+Currently, it is not OSS but part of our IPFS gateway API set will be OSS.
+https://github.com/filecoin-project/community/discussions/147
+
+## Website
+<!--Link your website if available-->
+https://ipfs-souko.nonentropy.jp/
```

Comment:
```suggestion
[IPFS Souko](https://ipfs-souko.nonentropy.jp/)
```

Reply:
@longfeiWan9 
Thanks for the suggestion. I have corrected it.